EmailSender.Api/appsettings.Debug.json

```diff
@@ -1,7 +1,14 @@
 {
+  "GmailOptions": {
+    "Host": "***secret***",
+    "Port": "***secret***",
+    "FromEmail": "***secret***",
+    "FromPassword": "***secret***"
+  },
   "SendGridOptions": {
     "SendGridAPIKey": "***secret***",
     "SenderEmail": "***secret***",
     "SenderName": "***secret***"
-  }
+  },
+  "SwitchOptions": "GmailOptions"
```

Comment:
No, remove this flag. Choose google smtp options if the app is launched in debug or development mode